docs/releases/1.8.3.txt

```diff
@@ -59,3 +59,5 @@ Bugfixes
   results (:ticket:`24924`).
 
 * Fixed usage of transforms in subqueries (:ticket:`24744`).
+* Fixed prefetch_related for models using UUID primary keys on non-PostgreSQL
```

Comment:
```
``prefetch_related()``
```
"on databases other than PostgreSQL" will allow you to avoid adding a word to the spelling list.